project_milestone/models/project.py

```diff
@@ -10,3 +10,10 @@ class Project(models.Model):
         "project.milestone", "project_id", string="Milestones", copy=True
     )
     use_milestones = fields.Boolean(help="Does this project use milestones?")
+
+    milestones_required = fields.Boolean()
+
+    @api.onchange("use_milestones")
+    def use_milestones_onchange(self):
```

Comment:
Onchange methods must be private, and unless there's a major reason, follow the standard of prefixing it with `_onchange_`. So it'd be something like:

```suggestion
    def _onchange_use_milestones(self):
```

Reply:
@yajo I will go with changing the method names. Thanks for the suggestions. These are done.